### ISSUE TITLE
Getting Vulnerability resolve error in docker image Debian Security Update for krb5 (DSA 5286-1) 

### ISSUE THREAD

**Issue by sanjeev02saraswat** [docker/desktop-linux#108]
- [x] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Actual behavior
I am getting azure docker image vulnerability <b>Debian Security Update for krb5 (DSA 5286-1)</b>
### Information

  - Linux distro:
  - Distro version:
  - Docker Desktop Version:

### Steps to reproduce the behavior

  1. I have .net Core Docker image and it was running properly but now i am getting error <b>Debian Security Update for krb5 (DSA 5286-1)</b> from azure team.
  2. I have upgraded the image but still getting this vulnerbility kindly suggest how to resolve this issue.
  3. Below is my dockerfile:
  FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
WORKDIR /app
EXPOSE 8080
ENV ASPNETCORE_URLS=http://*:8080

RUN apt-get update && apt-get upgrade -y
RUN apt-get install -y curl
RUN apt-get install -y libpng-dev libjpeg-dev curl libxi6 build-essential libgl1-mesa-glx
RUN curl -sL https://deb.nodesource.com/setup_lts.x | bash -
RUN apt-get install -y nodejs  

#RUN apt-get install -y nodejs
RUN groupadd --system --gid 1000 arucosmq \
    #Create user 'arumq' and add the user to the group 'arucosmq'    
    && useradd -m --uid 1000 --gid arucosmq arumq \
    #Add user=arumq to group=arucosmq:
    && usermod -aG arucosmq arumq 

FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
RUN apt-get update && apt-get upgrade -y
RUN apt-get install -y curl
RUN apt-get install -y libpng-dev libjpeg-dev curl libxi6 build-essential libgl1-mesa-glx
RUN curl -sL https://deb.nodesource.com/setup_lts.x | bash -
RUN apt-get install -y nodejs

#RUN apt-get install -y nodejs
RUN npm config set registry <registrypath>
RUN npm config set legacy-peer-deps true

COPY Optum_Root_CA.cer /usr/share/ca-certificates/
RUN echo Optum_Root_CA.cer >> /etc/ca-certificates.conf
RUN update-ca-certificates

WORKDIR /src
COPY ["WebAPP.csproj", "./"]
RUN dotnet restore "WebAPP.csproj"
COPY . .
WORKDIR "/src/."
RUN dotnet build "WebAPP.csproj" -c Release -o /app/build

FROM build AS publish
RUN dotnet publish "WebAPP.csproj" -c Release -o /app/publish

FROM base AS final
WORKDIR /app
COPY --from=publish /app/publish .

USER 1000

ENTRYPOINT ["dotnet", "WebAPP.dll"]

**Comment by p1-0tr** [comment]
hi @sanjeev02saraswat, thanks for filing your issue. Could you provide more information, like what OS and Docker Desktop version are you using, and ideally run diagnostics (https://docs.docker.com/desktop/troubleshoot/overview/) and provide a diagnostic id?

**Comment by sanjeev02saraswat** [comment]
Hello @p1-0tr
Hello @p1-0tr ,
We are using docker version 20.10.12. We have created image using above shared docker file and there is no issue on image creation. We are just getting vulnerability in deployed image and below is the vulnerability. Where as we have created new image still we are getting this vulnerability kindly suggest what we are doing wrong here.
Vulnerability:  <b>Debian Security Update for krb5 (DSA 5286-1)</b>
